prime-router/src/main/kotlin/fhirengine/engine/FHIRTranslator.kt

```diff
@@ -108,22 +108,30 @@ class FHIRTranslator(
     internal fun getByteArrayFromBundle(
         receiver: Receiver,
         bundle: Bundle,
-    ) = when (receiver.format) {
-        Report.Format.FHIR -> {
-            if (receiver.schemaName.isNotEmpty()) {
-                val transformer = FhirTransformer(receiver.schemaName)
+    ): ByteArray {
+        if (receiver.enrichmentSchemaNames.isNotEmpty()) {
+            receiver.enrichmentSchemaNames.forEach { enrichmentSchemaName ->
+                val transformer = FhirTransformer(enrichmentSchemaName)
```

Comment:
We should have a .info message here probably along the lines of "Applying schema <schema-name>". This will make more sense and be more helpful once we can add context to the log messages, like report_id.